### ISSUE TITLE
Mask TextSpotter: An End-to-End Trainable Neural Network for Spotting Text with Arbitrary Shapes

### ISSUE THREAD

**Issue by junxnone** [junxnone/tio#750]
# Reference
- [ ] [paper - 2018 - Mask TextSpotter: An End-to-End Trainable Neural Network for Spotting Text with Arbitrary Shapes](https://arxiv.org/pdf/1807.02242.pdf)
- [ ] [ECCV 2018 | 旷视科技提出新型端到端可训练网络Mask TextSpotter：可检测和识别任意形状的文本](https://www.sohu.com/a/242862623_418390)
- [ ] [Mask TextSpotter: An End-to-End Trainable Neural Network for Spotting Text with Arbitrary Shape](https://blog.csdn.net/weixin_43217144/article/details/84557191)
- [ ] [文本检测+识别之Mask TextSpotter](https://blog.csdn.net/qq_14845119/article/details/81505673)
- [ ] [图像文字识别初探(三)-Mask TextSpotter](https://blog.csdn.net/weixin_42111770/article/details/84881473)
- [ ] [文本检测之MaskTextSpotter](https://zhuanlan.zhihu.com/p/44491270)
- [ ] [MaskTextSpotter - Pytorch implementation - Github](https://github.com/MhLiao/MaskTextSpotter)

**Comment by junxnone** [comment]
#749
